.github/cspell.json

```diff
@@ -9,6 +9,13 @@
     "dictionarydefinitions": [],
     "dictionaries": [],
     "words": [
-        "ccnp"
+        "ccnp",
+        "TDVM",
+        "kustomization",
+        "srwxr",
+        "udev",
+        "udevadm",
+        "daemonset",
```

Comment:
Small comment: please use alphabeta order for the words

Reply:
More words can be added in dictionary, for example,
(I'm not sure whether we need add both lowcase or uppercase words, please check the cspell.json rules)

daemonset
CCNP
ccnp
POD
NFD
tdx
TEE
VM

Reply:
for more words, let's add and merge when our PRs get's in and the case is fine as per my check result